### DISCUSSION TITLE
Solax EV Charger without inverter

### DISCUSSION THREAD

**Discussion by maniakum** [PatrikTrestik/homeassistant-solax-http#43] [General]
Hi, I only have a solax charger which I connected via a RS485 to TCP/IP converter, but I can't get the data from the charger.

Is this possible, do you have any specific converter settings etc.? I couldn't find anything anywhere and I would like to get it working in HA to have data from the charger.

**Comment by PatrikTrestik** [comment]
Hi.
If you are connecting through RS485, you use Modbus protocol to communicate. 
This integration is using HTTP communication on local IP (where Web UI of EVC is accesible).

If you want to use Modbus, please look here https://github.com/wills106/homeassistant-solax-modbus there is planty of discussions how to setup. I can answer questions there too.
It is better to keep discussions where it belongs.

If you want to use HTTP connection, feel free to reopen this.

**Reply by maniakum** [reply]
I apologize. I will throw the question in the other forum. However, regarding the solax http integration, I was unable to set it up either.

1) I don't know where to find the IP address on the charger - I just tested the addresses assigned by the router
2) does it work just by entering the IP of the charger and the SN and it should go?

I couldn't get it to work, so I turned to Modbus.

**Comment by PatrikTrestik** [comment]
You can test IP by entering http://xxx.xxx.xxx.xxx to your browser. EVC UI should apear if you have correct IP.
You can test password (Not SN, but Registration number. Solax has switched terminology on several places) by logging into UI. RN is the password (if it was not changed previously).

**Comment by maniakum** [comment]
It worked. The only problem was that I entered the SN instead of the Registration Number, as it is indicated in the integration. It would probably be nice to rename it or put "SN/Registration number" there.

But it works. Thank you for your help.

**Comment by PatrikTrestik** [comment]
Sorry. It is my bug. I had impression that I already put explanation there, but it is not commited.
I'll fix it as soon as I'm by computer.
